src/NumField/Embedded.jl

```diff
@@ -323,6 +323,15 @@ function (QQ::QQField)(x::EmbeddedNumFieldElem)
   return QQ(data(x))
 end
 
+function Float64(x::EmbeddedNumFieldElem)
```

Comment:
```suggestion
function Base.Float64(x::EmbeddedNumFieldElem)
```
see https://github.com/thofma/Hecke.jl/pull/1742

Reply:
Thanks, should be added now!